devops/tasks.py

```diff
@@ -209,3 +224,198 @@ def release(
         logger.info(f"Removing temporary configurations from {rel_path}")
         if rel_path.exists():
             rmtree(rel_path)
+
+
+def get_master_key(env: str) -> None:
+    """
+    Get the master key for SealedSecrets for the given env.
+
+    :param str env: The environment
+    """
+    settings = load_env_settings(env)
+    ensure_context(settings.KUBE_CONTEXT)
+
+    label(logger.info, f"Getting master key for {env}")
+
+    # Based on:
+    # https://github.com/bitnami-labs/sealed-secrets#how-can-i-do-a-backup-of-my-sealedsecrets
+    result = run(
+        [
+            "kubectl",
+            "get",
+            "secret",
+            "-n",
+            "kube-system",
+            "-l",
+            "sealedsecrets.bitnami.com/sealed-secrets-key",
+            "-o",
+            "yaml",
+        ]
+    )
+
+    content = result.stdout.decode(encoding="utf-8")
+    output_file = master_key_path(env=env)
+
+    logger.info(f"Saving master key to {output_file}")
+
+    with output_file.open("w", encoding="utf-8") as f:
+        f.write(content)
+
+
+def unseal_secrets(env: str) -> None:
+    """
+    Decrypts the secrets for the desired env and base64 decodes them to make
+    them easy to edit.
+
+    :param str env: The environment.
+    """
+    # Validate env
+    load_env_settings(env)
+
+    master_key = master_key_path(env=env)
+    if not master_key.exists():
+        get_master_key(env=env)
+
+    sealed_secret_files = [
+        secret_file
+        for secret_file in (Path("envs") / env / "secrets").glob("*.yaml")
+        if not secret_file.name.endswith(UNSEALED_SECRETS_EXTENSION)
+    ]
+
+    label(logger.info, f"Unsealing secrets for {env}")
+
+    for input_file in sealed_secret_files:
+        output_file = input_file.with_name(input_file.stem + UNSEALED_SECRETS_EXTENSION)
+
+        logger.info(f"Unsealing {input_file} to {output_file}")
+
+        with input_file.open("r", encoding="utf-8") as f:
+            content = f.read()
+
+        content = kube_unseal(content, master_key)
+        content = base64_decode_secrets(content)
+
+        with output_file.open("w", encoding="utf-8") as f:
+            f.write(content)
+
+        input_file.unlink()
```

Comment:
Do we really want to delete the sealed files?

Reply:
Probably better to leave the files than unexpectedly delete them. The .gitgnore should hopefully avoid accidentally committing the files to the repo.